discovery/seata-discovery-namingserver/src/main/java/org/apache/seata/discovery/registry/namingserver/NamingserverRegistryServiceImpl.java

```diff
@@ -156,6 +156,9 @@ public void register(Instance instance) throws Exception {
         doRegister(instance, getNamingAddrs());
     }
 
+    public void doRegister(List<Instance> instance, List<String> urlList) {
+
```

Comment:
why this method is not implemented?

Reply:
This is a reserved interface because, when multi-raft is implemented, a node will have multiple instances.